### DISCUSSION TITLE
Remove initial settings screen?

### DISCUSSION THREAD

**Discussion by rp1231** [File-New-Project/EarTrumpet#738] [Ideas]
![EarTrumpet_X7rnYgzKE1](https://user-images.githubusercontent.com/47380919/107889634-08b81480-6f3a-11eb-8a27-709f6d340948.png)

This screen doesn't really add any value and adds an extra click.

I know that this is just a minor nitpick as I love eartrumpet but I think a small thing like this would make sense and iron out a small teeny tiny crease.

Thanks

**Comment by davuxcom** [comment]
Addons get a category in Settings.

![image](https://user-images.githubusercontent.com/1456216/107891119-87c53280-6ed1-11eb-8c20-7b6a751bc9b5.png)

It was considered to skip the click when no other categories are available, but we didn't do that work and don't intend to.

**Reply by rp1231** [reply]
Thanks for the reply.
Is this a newer build you're using?
I'm using the one from chocolatey and it doesn't have the addons option and all the settings in your screenshot.

**Reply by davuxcom** [reply]
Addons haven't shipped yet, but since you asked why it is the way it is, so I explained.

**Reply by rp1231** [reply]
Oh ok,
Thanks
